src/Benchmarks/Configuration/AppSettings.cs

```diff
@@ -7,5 +7,6 @@ namespace Benchmarks.Configuration
     public class AppSettings
     {
         public string ConnectionString { get; set; }
+        public string Database { get; set; } = "sqlserver";
```

Comment:
I would prefer to use an enum rather than strings.  Create a enum named something like `Database` with values `SqlServer` and `PostgreSQL`.  Then change this property in AppSettings from `string` to `Database`, and the framework will handle converting the string in `appsettings.json` to the correct enum value.